lib/resources/example_json/r4_examples_person.rb

```diff
@@ -0,0 +1,323 @@
+module Cerner
+    module Resources
+  
+      R4_COMBINED_PERSON_ENTRY ||= {
+        "resourceType": "Person",
+        "id": "971923",
+        "meta": {
+            "versionId": "0"
+        },
+        "text": {
+            "status": "generated",
+            "div": "&lt;div xmlns=\"http://www.w3.org/1999/xhtml\"&gt&lt;p&gt&lt;b&gtPerson&lt;/b&gt&lt;/p&gt&lt;p&gt&lt;b&gtStatus&lt;/b&gt: Inactive&lt;/p&gt&lt;p&gt&lt;b&gtLink&lt;/b&gt: Person/933924&lt;/p&gt&lt;/div&gt"
```

Comment:
All of the end tags in all of these examples are not rendering correctly

Reply:
Corrected in commit 6b67bcc19cfed472c7ec020f38e7d45ab0067dcc